### ISSUE TITLE
Synthetic Scene Results

### ISSUE THREAD

**Issue by ashmrz** [fudan-zvg/4d-gaussian-splatting#29]
Hello. I experimented with the synthetic scenes, and I'd like to share an example result I obtained on the T-Rex scene. I noticed variations in the parameters used in the configuration files for some scenes, including the T-Rex scene, compared to what was reported in the paper. Could you please review and advise if there might be any oversight on my part in running the code? Additionally, I achieved a PSNR of 29.88 dBs after training on the test set. Is this consistent with what you get? 

<img width="779" alt="Screen Shot 2024-03-13 at 9 33 29 AM" src="https://github.com/fudan-zvg/4d-gaussian-splatting/assets/31591367/7cd6e9f6-1b69-49e4-b6b8-00358751ae93">

**Comment by Alexander0Yang** [comment]
Hi @ashmrz ,
In some synthetic scenes, overfitting can happen if proper regularization is not applied. Thus, the best PSNR might be occurred at the early stage of training. Specifically, in the T-rex scene, we achieved a PSNR of 31.29 at the 4000th iteration.

**Comment by kevinliu-ez** [comment]
hello ashmrz: do you write the .pth viewer?
